test/jdk/java/foreign/TestScope.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, 2024 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Can you hold off on this until TestScope.java's copyright header is fixed in main line.

Reply:
This has been fixed now.